<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.3 (Python 3.9.5 on win32)" generated="20211219 00:08:49.863" rpa="false" schemaversion="2">
<suite id="s1" name="meuTest" source="C:\robotFrameWork\testCase1\meuTest.robot">
<test id="s1-t1" name="CT001">
<kw name="Primeira keyword" library="myResources">
<kw name="Log To Console" library="BuiltIn">
<arg>${\n}Primeiro teste com o robot executado com sucesso!</arg>
<doc>Logs the given message to the console.</doc>
<status status="PASS" starttime="20211219 00:08:49.918" endtime="20211219 00:08:49.926"/>
</kw>
<status status="PASS" starttime="20211219 00:08:49.918" endtime="20211219 00:08:49.926"/>
</kw>
<status status="PASS" starttime="20211219 00:08:49.917" endtime="20211219 00:08:49.927"/>
</test>
<status status="PASS" starttime="20211219 00:08:49.867" endtime="20211219 00:08:49.931"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="meuTest">meuTest</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20211219 00:08:49.909" level="ERROR">Error in file 'C:\robotFrameWork\testCase1\meuTest.robot' on line 4: Importing library 'SeleniumLibary' failed: ModuleNotFoundError: No module named 'SeleniumLibary'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\robotFrameWork\robot\Scripts\robot.exe
  C:\Python39\python39.zip
  C:\Python39\DLLs
  C:\Python39\lib
  C:\Python39
  c:\robotframework\robot
  c:\robotframework\robot\lib\site-packages</msg>
</errors>
</robot>
